package.json

```diff
@@ -81,7 +81,7 @@
     "moment-strftime": "~0.5.0",
     "moment-timezone": "~0.5.40",
     "numeral": "~2.0.6",
-    "patternfly": "~3.59.1",
+    "patternfly": "~3.59.5",
```

Comment:
This might be a significant change.  Did you click through the UI and verify?

Reply:
Here's the changes: https://github.com/patternfly/patternfly-3/compare/v3.59.1...v3.59.5

3.59.5 came out in 2020 - I'm surprised we weren't on the latest.  So I'm 100% on board with bumping, but we should just verify nothing breaks in doing so.

Reply:
Clicked through and didn't see anything breaking or acting strangely